backend/routers/ai.py

```
@@ -0,0 +1,169 @@
+from fastapi import APIRouter, Depends, HTTPException, status, Request
```

Ruff (0.11.9)

`fastapi.HTTPException` imported but unused

Remove unused import

(F401)